ptvsd/pathutils.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root
+# for license information.
+
+from __future__ import print_function, with_statement, absolute_import
```

Comment:
No need for `with_statement` (it's from the days when we had to support Python 2.5 etc).